graphqlapi/src/main/java/io/stargate/graphql/schema/FieldTypeCache.java

```diff
@@ -0,0 +1,95 @@
+package io.stargate.graphql.schema;
```

Comment:
Should these 4 classes maybe go under a `cache` package?

Reply:
Yes, the root package is starting to get crowded, I was thinking of some reorganizing. Maybe something like `schema/dml` and `schema/ddl`, each with their own `fetchers` subpackage and maybe others.